virttest/cpu.py

```diff
@@ -848,9 +848,12 @@ def _make_up_pattern(flags):
     cpu_types = {"AuthenticAMD": ["EPYC-Rome", "EPYC", "Opteron_G5",
                                   "Opteron_G4", "Opteron_G3", "Opteron_G2",
                                   "Opteron_G1"],
-                 "GenuineIntel": ["KnightsMill", "Icelake-Server",
-                                  "Icelake-Client", "Cascadelake-Server",
-                                  "Skylake-Server", "Skylake-Client",
+                 "GenuineIntel": ["KnightsMill", "Cooperlake",
+                                  "Icelake-Server", "Icelake-Server-noTSX",
+                                  "Icelake-Client", "Icelake-Client-noTSX",
+                                  "Cascadelake-Server", "Cascadelake-Server-noTSX",
+                                  "Skylake-Server", "Skylake-Server-noTSX-IBRS",
```

Comment:
Would you pls explain why add "Skylake-Server-noTSX-IBRS" other than "Skylake-Server-noTSX"? I would prefer to keep consistent with other cpu models.

Reply:
There is no "Skylake-Server-noTSX" model in qemu5.0, please help check again, thanks.